osu.Game/Graphics/UserInterface/DialogButton.cs

```diff
@@ -13,30 +13,41 @@
 using osu.Game.Graphics.Backgrounds;
 using osu.Game.Graphics.Sprites;
 
-namespace osu.Game.Overlays.Pause
+namespace osu.Game.Graphics.UserInterface
 {
-    public class PauseButton : ClickableContainer
+    public class DialogButton : ClickableContainer
     {
         private const float hover_width = 0.9f;
         private const float hover_duration = 500;
         private const float glow_fade_duration = 250;
         private const float click_duration = 200;
 
-        private Color4 backgroundColour = OsuColour.Gray(34);
-
-        private Color4 buttonColour;
-        public Color4 ButtonColour
+        private Color4 colour;
+        public new Color4 Colour

```

Comment:
i think we're avoiding `new` overrides like this at the moment (still not 100% sure it's the best way) so ButtonColour is probably best here.